heltour/tournament/tasks.py

```diff
@@ -311,6 +317,99 @@ def update_slack_users():
             p.save()
 
 
+def _start_league_games(*, tokens, clock, increment, do_clockstart, clockstart, clockstart_in, variant, leaguename, league_games):
+    try:
+        result = lichessapi.bulk_start_games(tokens=tokens, clock=clock, increment=increment, do_clockstart=do_clockstart, clockstart=clockstart, clockstart_in=clockstart_in, variant=variant, leaguename=leaguename)
+    except lichessapi.ApiClientError as err:
+        # try to handle errors due to rjected tokens
+        e = str(err).replace('API failure: CLIENT-ERROR: [400] ', '') # get json part from error
+        try:
+           result = json.loads(e)
+           for bad_token in result["tokens"]:
+               # set expiration of rejected token to yesterday, so we know to not use it anymore.
+               for game in league_games:
+                   if game.white.oauth_token.access_token==bad_token:
```

Comment:
I feel like his inner loop and check should be refactored out to helper methods to make this easier to follow.